### DISCUSSION TITLE
JULES Documentation and Useful links

### DISCUSSION THREAD

**Discussion by richardgilham** [MetOffice/simulation-systems#67] [JULES]
**Working Practices:** https://metoffice.github.io/simulation-systems/
**GitHub:** https://github.com/MetOffice/jules
**User Guide:** https://metoffice.github.io/jules
**JULES community website:** https://jules.jchmr.org

**[Frozen] Documentation:** https://jules-lsm.github.io/latest/index.html

**[Frozen] TRAC:** https://code.metoffice.gov.uk/trac/jules/wiki (MOSRS login required) 

Please drop any suggestions for further links as a comment as we can edit this pinned post.

**Comment by soi521** [comment]
Please add the JULES website https://jules.jchmr.org/ - Thanks! Toby
